cmd/protoc-gen-micro/plugin/micro/micro.go

```diff
@@ -137,6 +137,11 @@ func (g *micro) generateService(file *generator.FileDescriptor, service *pb.Serv
 	}
 	servName := generator.CamelCase(origServName)
 	servAlias := servName + "Service"
+	fullServName := servName
+	useGrpc := g.gen.Param["use_grpc"]
```

Comment:
useGrpc := g.gen.Param["use_grpc"] != ""
Do you think it's better to define userGrpc as bool?

Reply:
Originally, I thought I only needed this parameter, but it must be assigned when executing the command, so I wrote use_ grpc = 1, in fact, any value is OK.
However, since the assignment is made, it is better to have a clearer range of values. I'll modify it later.

Reply:
All right, I get it.